### PR TITLE
Supplementary Docs

### PR DESCRIPTION
**Pull request by JimBobSquarePants**
Fleshing out supplamentary docs

- [x] Getting Started
- [x] Processing

### CODE REVIEW

File: articles/ImageSharp/GettingStarted.md [hunk 1]
```diff
@@ -1,37 +1,52 @@
...
+} // Dispose - releasing memory into a memory pool ready for the next image you wish to process.
 ```
 
 ### API Style
 The easiest way to work with ImageSharp is to utilize our extension methods:
-- @"SixLabors.ImageSharp.ImageExtensions" for basic stuff
-- @"SixLabors.ImageSharp.Processing.ProcessingExtensions" for `Mutate()` and `Clone()` 
+- @"SixLabors.ImageSharp" for basic stuff.
+- @"SixLabors.ImageSharp.Processing" for `Mutate()` and `Clone()` 
 - All the processing extensions (eg. `Resize(...)`) are living within the @"SixLabors.ImageSharp.Processing" namespace.
```
Comment (tocsoft) [inline articles/ImageSharp/GettingStarted.md]:
```suggestion
 - All the processing extensions (eg. `Resize(...)`) live within the @"SixLabors.ImageSharp.Processing" namespace. 
```

File: articles/ImageSharp/GettingStarted.md [hunk 2]
```diff
@@ -1,37 +1,52 @@
...
 
 ### API Style
 The easiest way to work with ImageSharp is to utilize our extension methods:
-- @"SixLabors.ImageSharp.ImageExtensions" for basic stuff
-- @"SixLabors.ImageSharp.Processing.ProcessingExtensions" for `Mutate()` and `Clone()` 
+- @"SixLabors.ImageSharp" for basic stuff.
+- @"SixLabors.ImageSharp.Processing" for `Mutate()` and `Clone()` 
 - All the processing extensions (eg. `Resize(...)`) are living within the @"SixLabors.ImageSharp.Processing" namespace. 
-If you want to do image processing work, make sure you are `using` this namespace in your code!
+If you want to do image processing work, make sure you are `using` this namespace in your code.
```
Comment (tocsoft) [inline articles/ImageSharp/GettingStarted.md]:
```suggestion
If you want to do image processing work, make sure you add `using SixLabors.ImageSharp.Processing;`.
```

File: articles/ImageSharp/PixelFormats.md [hunk 3]
```diff
@@ -2,9 +2,10 @@
...
-A pixel is basically a small value object (struct), describing the color at a given point according to a pixel model we call Pixel Format / Pixel Type. `Image<TPixel>` represents pixel graphic bitmap stored as a **generic, contigous memory block** of pixels, having the size `image.Width * image.Height`. 
-In case of multi-frame images (usually decoded from gifs) multiple bitmaps are stored in `image.Frames` as `ImageFrame<TPixel>` instances.
+We support multiple pixel formats just like *System.Drawing* does. However, unlike their closed [PixelFormat](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.imaging.pixelformat) enumeration, our solution is extensible.
+A pixel is basically a small value object (struct), describing the color at a given point according to a pixel model we call Pixel Format / Pixel Type. `Image<TPixel>` represents a pixel graphic bitmap stored as a **generic, contiguous memory block** of pixels, having the size `image.Width * image.Height`.
```
Comment (tocsoft) [inline articles/ImageSharp/PixelFormats.md]: 'Pixel Format / Pixel Type' : we should pick just one and stick to only that name to avoid confusion.

Comment (tocsoft) [inline articles/ImageSharp/PixelFormats.md]: do we need the word 'generic' in 'generic, contiguous memory block'?

File: articles/ImageSharp/Processing.md [hunk 4]
```diff
@@ -1 +1,48 @@
-TODO
\ No newline at end of file
+# Processing
+
+The ImageSharp processing API is imperative. This means that the order in which you supply the individual processing operations is the order in which they are are compiled and applied. This allows the API to be very flexible, allowing you to combine processes in any order.
+
+Processing operations are implemented using one of two available method calls. 
+[`Mutate`](xref:SixLabors.ImageSharp.Processing.ProcessingExtensions.Mutate*?displayProperty=name) and [`Clone`](xref:SixLabors.ImageSharp.Processing.ProcessingExtensions.Clone*?displayProperty=name)
+
+The difference being that the former applies the given processing operations to the current image whereas the latter applies the operations to a depp copy of the original image.
```
Comment (tocsoft) [inline articles/ImageSharp/Processing.md]:
```suggestion
The difference being that the former applies the given processing operations to the current image whereas the latter applies the operations to a deep copy of the original image.
```

File: articles/ImageSharp/GettingStarted.md [hunk 5]
```diff
@@ -1,37 +1,52 @@
...
 
 ### Why is [](xref:SixLabors.ImageSharp.Image`1?displayProperty=name) a generic class?
 An image is essentially a **generic 2D array of pixels** stored in a contiguous memory block. Check out the [Pixel Formats](PixelFormats.md) article for more details!
```
Comment (tocsoft) [inline articles/ImageSharp/GettingStarted.md:52]: is the word 'generic' needed?

**Review by tocsoft — COMMENTED** [review]
just some initial things i've spotted/questioning

**Comment by SimonCropp** [comment]
was gong to do a docs PR. but i think i will wait for this one to finish :)

**Comment by JimBobSquarePants** [comment]
I've made some updates and am going to merge as-is. We can keep chipping away.

**Pull request merged**